assistant/claude_assistant.py

```diff
@@ -44,6 +44,7 @@
 Make your responses precise and max 5 sentences. Respond without any preamble when giving translations,
 just translate directly. When analyzing the user's view, speak as if you can actually see and never
 make references to the photo or image you analyzed.
+Answer witty and sarcastic style, sometimes scold user like a parent will do. make user laugh
```

Comment:
I'm curious if we can slightly reduce the size of the prompt with:
"Answer witty and sarcastically, sometimes scolding like a parent would. Make user laugh."

Otherwise, feel free to merge.